docs/gambit/mutation-verifier.md

```diff
@@ -36,52 +36,74 @@ Once you have updated your `certora-cli` installation using `pip` to get the rel
 dependencies, run Gambit from the command line:
 
 ```
-certoraMutate path/to/config/file/Example.conf
+certoraMutate --prover_conf path/to/prover/config.conf --gambit_conf path/to/gambit/config.conf
 ```
 
 (gambit-prover-config)=
 ## Configuration
-The tool expects a configuration file (extension `.conf` is required).
-which is a JSON object.
-Here is an example configuration file:
+The tool expects two separate configuration files (extension `.conf` is required for both):
+the configuration file which defines the execution of mutant generation (`--gambit_conf`),
+and the configuration file which defines execution of the prover (`--prover_conf`).
+Here is a simple configuration file setup:
+
+In prover.conf:
+
+```json
+{
+  "files": [
+    "C.sol"
+  ] ,
+  "process": "emv",
+  "prover_args": [
+    " -adaptiveSolverConfig false -smt_nonLinearArithmetic false"
+  ],
+  "solc": "solc8.1",
+  "verify": "C:c.spec"
+}
+```
+In gambit.conf:
+
+# In prover.conf:
+```json
+{
+  "filename" : "Test/10Power/TenPower.sol",
+  "solc" : "solc8.10",
+  "num_mutants": 5
+}
+```
+
+### Auto-configuration of Mutant Generation
+
+Note: The most common use case of `certoraMutate` is to run on a project that has already been verified by `certoraRun`.
+Therefore, it is recommended to reuse the prover configuration file from verification for mutation testing. For
+convenience, the `--auto_conf` flag is available. When present, a configuration json for mutant generation is
+automatically generated from the prover configuration (the argument to `--prover_conf`) and written to the
+file provided as an argument to `--gambit_conf`. This allows the user to fine-tune the configuration without having to
+write any additional boilerplate from scratch.
+
+### Manual Mutations
+
+In addition to the prover and mutant generation configuration files, the optional flag
+`--manual_mutations` is supported. This allows the user to supplement Gambit's mutant generation operations with
+manually-written mutants. This flag expects as an argument a json file which describes the manual mutations to include.
+Here is an example of such a file:
 
 ```json
 {
-    "project_folder" : "Test/10Power",
-    "run_script" : "Test/10Power/runDefault.sh",
-    "gambit" : {
-        "filename" : "Test/10Power/TenPower.sol",
-        "solc" : "solc8.10"
-    },
-    "staging" : "master",
-    "manual_mutations" : {
-        "path/to/original/file.sol" : "path/to/manual/mutations/",
-        "...": "..."
-    },
-    "...": "..."
+  "C.sol": [
```

Comment:
I dont think we should do manual mutations this way. Let's chat on the other PR.